clang/lib/CodeGen/CGClass.cpp

```diff
@@ -2779,6 +2779,39 @@ void CodeGenFunction::EmitTypeMetadataCodeForVCall(const CXXRecordDecl *RD,
   }
 }
 
+/// Converts the CFITypeCheckKind into SanitizerKind::SanitizerOrdinal and
+/// llvm::SanitizerStatKind.
+static std::pair<SanitizerKind::SanitizerOrdinal, llvm::SanitizerStatKind>
+ParseCFITypeCheckKind(CodeGenFunction::CFITypeCheckKind TCK) {
```

Comment:
(optional) I don't really like the function name, because it doesn't really parse anything per se

Reply:
SanitizerInfoFromCFICheckKind? Also not great.

Reply:
Done